crates/ruma-events/src/lib.rs

```diff
@@ -194,6 +194,7 @@ pub use self::{
 
 /// Extra information about an event that is not incorporated into the event's
 /// hash.
+#[allow(clippy::exhaustive_structs)]
 #[derive(Clone, Debug, Default, Deserialize, Serialize)]
 pub struct Unsigned {
```

Comment:
I think this one should actually be non-exhaustive (and have `new` constructor that delegates to `Default`).